### PR TITLE
feat: Adds FeaturesNodeConnected component. 

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

Adds a FeaturesNodeConnected component. This PR also make a number of changes and tweaks to the `ui-connected` story setup.  I added a `apollos-stories` package which allows us to easily mock Apollo data in stories. This will make it far simpler to write stories for connected components, which is going to be very important. 

Simulator Screen Shot - iPhone 11 - 2020-09-02 at 12 21 24

### What design trade-offs/decisions were made?

The new apollo storybook mock requires us to guess at what the schema is. Rather than keep this file "synced" from the apollos-church-api project, I have copied it in. I think that's a fair assumption to make, since we don't care too much about the specifics  of the schema, merely that it resembles a client church's schema. The schema "up to dateness" will be enforceable because we won't be able to write stories for any new parts of the schema without copying in the most recent version of the schema. 

### How do I test this PR?

You know the drill :) Grab the companion PR here: https://github.com/ApollosProject/apollos-templates/pull/94/files

From there, you can open up storybook and visit the FeaturesNodeConnected story. 

### What automated tests did you write?

New snapshots.

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-data-schema/src/index.js [hunk 1]
```diff
@@ -25,10 +25,6 @@ export const interfacesSchema = gql`
     audios: [AudioMedia]
   }
 
-  interface FeaturesNode {
```
Comment (vinnyjth) [inline packages/apollos-data-schema/src/index.js:28]: Moved this to the Features section, which is how I've handled other "optional" interfaces.

File: packages/apollos-server-core/src/interfaces/resolver.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default {
   LiveNode: {
     __resolveType,
   },
-  CardNode: {
+  Card: {
```
Comment (vinnyjth) [inline packages/apollos-server-core/src/interfaces/resolver.js:14]: This was a typo from my last PR.

File: packages/apollos-ui-connected/src/ActionListFeatureConnected/ActionListFeature/ActionListFeature.tests.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 
-import { Providers } from '../../utils/testUtils';
+import { Providers } from '../../testUtils';
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ActionListFeatureConnected/ActionListFeature/ActionListFeature.tests.js:4]: I moved around the test utils, you'll see a lot of these changes.

File: packages/apollos-ui-connected/src/NodeFeaturesConnected/NodeFeaturesConnected.stories.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+storiesOf('ui-connected/NodeFeaturesConnected', module)
+  .addDecorator((story) => (
+    <BackgroundView>
+      {/* eslint-disable-next-line react-native/no-inline-styles */}
+      <CenteredView style={{ alignItems: 'stretch' }}>{story()}</CenteredView>
+    </BackgroundView>
+  ))
+  .addDecorator(ApolloStorybookDecorator({ mocks }))
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/NodeFeaturesConnected/NodeFeaturesConnected.stories.js:53]: here's our decorator :)

File: packages/apollos-ui-connected/src/features/ScriptureFeature.js [hunk 5]
```diff
@@ -10,10 +10,13 @@ const ScriptureFeature = ({
   sharing: { message } = {},
   isLoading,
   contentId,
+  nodeId,
 }) => (
   <ActionCard
     icon={'text'}
-    action={<ShareButtonConnected message={message} itemId={contentId} />}
+    action={
+      <ShareButtonConnected message={message} itemId={nodeId || contentId} />
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/features/ScriptureFeature.js:18]: Moving towards a future where ShareButtonConnected won't be `content` specific.

**Review by conrad-vanl — APPROVED** [review]

**Pull request merged**